### PR TITLE
Green banner

### PR DESCRIPTION
**Pull request by tomltd**

### CODE REVIEW

**Comment by tomltd** [comment]
Found some #start links and replaced with #categories then did a grep to see if there were any more. :zap:

**Comment by katjam** [comment]
Oh that's a good point. I think maybe start function used to
start-or-resume as in if started already goes to question, if not goes to
category page. We better check that.

**Comment by katjam** [comment]
I think it's ok to go straight to #categories in stead - but can you just check that the data-action=#start wasn't triggering anything ever?

also - you missed one - in the resume function scripts.js

**Comment by tomltd** [comment]
I think I've sorted it out now @katjam merge?

File: src/js/scripts.js [hunk 1]
```diff
@@ -341,9 +341,6 @@ function restart() {
   db.set('pipAss.category', null);
   db.set('pipAss.remainingCategories', _.uniq(window.allCategories));
 
-  // go to start screen
-  loadSlide('start');
-
```
Comment (katjam) [inline src/js/scripts.js]: I think this should be loadSlide('categories');

**Comment by katjam** [comment]
nearly :hamburger: :hand:

**Comment by tomltd** [comment]
I was searching for '#start' instead of just 'start' doh...

**Comment by katjam** [comment]
yup. That's it I think. :footprints:

**Pull request merged**